proto/api/v1/api.proto

```diff
@@ -799,3 +800,78 @@ option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
     }
   }
 };
+
+service PluginService {
+  rpc ListPlugins(ListPluginsRequest) returns (ListPluginsResponse) {
+    option (google.api.http) = {
+      get: "/v1/plugins"
+      response_body: "plugins"
+    };
+  };
+}
+
+message ListPluginsRequest {
+  // regex to filter plugins by name
+  string name = 1;
+}
+
+message ListPluginsResponse{
+  repeated PluginSpecifications plugins = 1;
+}
+
+message GetPluginRequest {
+  string name = 1;
+}
+
+message GetPluginResponse {
+  PluginSpecifications plugin = 1;
+}
+
+message PluginSpecifications{
```

Comment:
should I rename this to `ConnectorSpecifications` ?

Reply:
No, internally we still have to make a difference between plugins and connectors, they are distinct entities in Conduit.